setup.cfg

```diff
@@ -111,3 +112,55 @@ max-line-length = 79
 exclude =
     .git
     __pycache__
+
+[coverage:run]
+branch = False
+include =
+    odl/*
+omit =
+    */__init__.py
+    odl/diagnostics/*
+    odl/util/*
```

Comment:
Why do we omit this?

Reply:
I don't know. My guess is that it's from a time when the utility code was really only for internal usage and rather small in scope. Mostly little helpers and testing stuff. I'll check what makes sense to exclude there.

Reply:
Now more fine-grained. And `tests/` excluded, they used to drag down our coverage score.